taxcalc/params.json

```diff
@@ -930,6 +995,32 @@
         "col_label": "",
         "value": [0]
     },
+     "_ID_Mortgage_HC":{
+        "long_name": "Mortgage interest deduction haircut",
+	"description": "This percentage can be applied to limit the amount of mortgage interest deduction allowed.",
+        "irs_ref": "",
+        "notes": "This parameter is currently used to eliminate this part of itemized deduction. ",
+        "start_year": 2013,
+        "col_var": "",
+        "row_var": "",
+        "row_label": ["2013"],
+        "cpi_inflated": false,
+        "col_label": "",
+        "value":     [0.0] 
+    },
+    "_ID_DeductionBenefit_crt":{
+        "long_name": "Deduction tax benefit cap, ceiling as a percent of AGI",
+	"description": "This percentage is used to cap the tax benefit from deductions. Users can specify what deductions they want to apply to. ",
+        "irs_ref": "",
+        "notes": "",
+        "start_year": 2013,
+        "col_var": "",
+        "row_var": "",
+        "row_label": ["2013"],
+        "cpi_inflated": false,
+        "col_label": "",
+        "value":     [0.0] 
```

Comment:
This should be 1 in current law. Well, actually I'm interested to know whether there is any difference if this is 1 or 1e309.